LICENSE

```diff
@@ -0,0 +1,2 @@
+YEAR: 2018
+COPYRIGHT HOLDER: Ben Gready
```

Comment:
Thanks for adding this and the license. It's something I've basically not thought of much before. If it's acceptable to add more than one person, happy for you to be on here as well.